client/client.go

```diff
@@ -2,69 +2,76 @@ package client
 
 import (
 	"crypto/tls"
-	"encoding/json"
+	"encoding/base64"
 	"math/rand"
 	"net/http"
+	"net/url"
 	"os"
 	"strings"
 	"time"
 
+	gsclient "github.com/giantswarm/gsclientgen/client"
+	"github.com/giantswarm/gsclientgen/client/auth_tokens"
+	"github.com/giantswarm/gsclientgen/models"
+	"github.com/giantswarm/gsctl/client/clienterror"
 	"github.com/giantswarm/microerror"
+	httptransport "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/strfmt"
 	rootcerts "github.com/hashicorp/go-rootcerts"
-	gsclientgen "gopkg.in/giantswarm/gsclientgen.v1"
 )
 
 var (
 	// DefaultTimeout is the standard request timeout applied if not specified
-	DefaultTimeout time.Duration = 60 * time.Second
+	DefaultTimeout = 60 * time.Second
 
 	randomStringCharset = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
 
 	requestIDHeader string
 	cmdLine         string
 )
 
-// Configuration is the configuration to be used when creating a new API client
+func init() {
+	rand.Seed(time.Now().UnixNano())
+	requestIDHeader = randomRequestID()
+	cmdLine = getCommandLine()
```

Comment:
I would rather have `requestIDHeader` and `cmdLine` initialized in `NewV2` and possibly attached to `WrapperV2` as they belong to the request being made (e.g. `UserLogin`) rather than the package.

Reply:
Changed with the latest commit